### DISCUSSION TITLE
How do I delete a password store?

### DISCUSSION THREAD

**Discussion by cloakedape** [gopasspw/gopass#2279] [Q&A] [answered]
Hello,
how do I completey remove a password store?
I tried removing gopass directories from all paths but when I run `gopass setup` it brings up the old key and email.

**Comment by dominikschulz** [comment, marked as answer]
The GPG key is not part of gopass. It is part of your GPG keyring.

Use `gpg -K` to list secrets keys and `gpg –delete-secret-key KEYID` to remove the ones you don't need.
